src/controllers/routers/comments.js

```diff
@@ -28,7 +28,7 @@ const getCommentsbyCaseNumber = async (req, res, next) => {
       return next(new ResourceNotFound(`Case ${id} does not exist`));
     }
     const commentByCaseNumber = await Comment.query()
-      .select('message', 'author')
+      .select('caseNumber', 'message', 'author', 'createdAt')
```

Comment:
requesting these changes to the comments table:
* caseNumber to be a reference to case.id instead of to caseNumber
* author to be a reference to staff.id instead of a string

the response to the endpoint should include these info:
comment id, message, staff/author id, staff/author name, comment creation date